moonstone/plot/global_plot.py

```diff
@@ -0,0 +1,265 @@
+import pandas as pd
+import sys
+import typing
+from typing import Optional, List
+import warnings
+
+from moonstone.plot.plot_template import (
+    BarGraph,
+    Histogram
+)
+
+"""
+plot that could be used on any dataframe no matter the module
```

Comment:
Maybe what you set up in this all `global_plot` would actually the part of each `visualize` method from every module?